tng-router/config/sp_routes.yml

```diff
@@ -118,23 +118,32 @@ paths:
   /api/v3/slas/templates(/?|/*): 
     site: http://tng-sla-mgmt:8080/tng-sla-mgmt/api/slas/v1/templates
     permissions:
-    - role: '-'
-      verbs: get
+    - role: 'admin'
+      verbs: 'options,post,get,delete'
+    - role: 'customer'
+      verbs: 'options,get'
   /api/v3/slas/configurations(/?|/*): 
     site: http://tng-sla-mgmt:8080/tng-sla-mgmt/api/slas/v1/mgmt
     permissions:
     - role: '-'
-      verbs: get
+      verbs: 'options,get'
   /api/v3/slas/agreements(/?|/*): 
     site: http://tng-sla-mgmt:8080/tng-sla-mgmt/api/slas/v1/agreements
     permissions:
     - role: '-'
-      verbs: get
+      verbs: 'options,post,get,delete'
```

Comment:
are you sure you want any user to be able to do this?

Reply:
I gave it a second thought and I think you are right. It shouldn't have any user access to this...
Please check my update and let me know if you agree and this makes sense now.